superset-frontend/src/chart/Chart.jsx

```diff
@@ -185,14 +179,14 @@ class Chart extends React.PureComponent {
       errorMessage,
       onQuery,
       refreshOverlayVisible,
+      queriesResponse,
     } = this.props;
 
     const isLoading = chartStatus === 'loading';
-
     const isFaded = refreshOverlayVisible && !errorMessage;
     this.renderContainerStartTime = Logger.getTimestamp();
     if (chartStatus === 'failed') {
-      return this.renderErrorMessage();
+      return queriesResponse.map(item => this.renderErrorMessage(item));
```

Comment:
Do you have a screenshot of multiple errors rendered in a chart?

Reply:
Hi good catch, I didn't check it properly for all kinds of charts (new/old). I pushed now some small fix for it. Basically for now for new api there is no case that it returns more than 1 error per request even for multi queries, but if it will return we can support it, but for now it will be only one row.

Attaching screenshots after my fix for charts:
1. Old charts:
![Screen Shot 2020-12-03 at 9 02 11](https://user-images.githubusercontent.com/56388545/100975845-5205e400-3547-11eb-98d4-10b4daeddb9c.png)

2. New charts:
![Screen Shot 2020-12-03 at 9 10 54](https://user-images.githubusercontent.com/56388545/100975945-782b8400-3547-11eb-860b-aef2a754ca7a.png)

3. New charts with multiqueries (looks the same):
![Screen Shot 2020-12-03 at 9 00 19](https://user-images.githubusercontent.com/56388545/100975993-8b3e5400-3547-11eb-9e0d-3735901e756a.png)